packages/app/src/main.ts

```diff
@@ -1,6 +1,6 @@
 import '@fortawesome/fontawesome-free/css/all.css';
 import '../index.css';
-import '@fontsource/roboto'; // TODO disable font to get rid of an error
+import '@fontsource/roboto/700.css'; // TODO disable font to get rid of an error
```

Comment:
According to figma it should be 400 (regular
```suggestion
import '@fontsource/roboto/400.css'; // TODO disable font to get rid of an error
```